R/tm_a_pca.R

```diff
@@ -906,11 +910,12 @@ srv_a_pca <- function(id, data, reporter, filter_panel_api, dat, plot_height, pl
       teal::validate_inputs(iv_extra, header = "Plot settings are required")
 
       switch(input$plot_type,
```

Comment:
the same indentation case as for `geom_bar` and `if ` above

Reply:
auto lint revert back changes for switch statement https://github.com/insightsengineering/teal.modules.general/commit/7fac1eabd8686ff448741ac03a343a268cea15a8.